app/_includes/md/kgo/prerequisites.md

```diff
@@ -47,6 +47,17 @@ kubectl set env -n kong-system deployments/kgo-gateway-operator-controller-manag
 ```
 {% endif %}
 
+{%- if include.kongplugininstallation %}
+
+### Enable the KongPluginInstallation controller
+
+As this guide uses the experimental KongPluginInstallation feature, we need to explicitly enable it.
```

Comment:
After setting this env, KGO's Pod needs to be restarted for this to take effect, right?

Reply:
`Pod`s are restarted out of the box when `setenv` is used. Explicit restart is required in case when you have env vars from `ConfigMap`and you've updated its content

Reply:
Oh, nice. Wasn't aware of this. 👍